index.html

```diff
@@ -1,25 +1,31 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" href="style.css">
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css" />
         <title>Landing Page</title>
     </head>
 
     <body>
         <div class="header">
             <div class="top">
                 <h1>Header Logo</h1>
-                <a href="#"> header link one</a>
-                <a href="#"> header link two</a>
-                <a href="#"> header link three</a>
+                <nav class="nav">
+                    <a href="#"> header link one</a>
+                    <a href="#"> header link two</a>
+                    <a href="#"> header link three</a>
+                </nav>
             </div>
-            <div class="topHalfSplit">
+            <div class="top-half-split">
```

Comment:
CSS class names are idiomatically [kebab case](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case) because they can get quite long in which case kebab case is easier to read. 

Additionally, file names, URLs, HTML attributes, and many other web strings are kebab case.

And finally, CSS classnames are case insensitive, so it removes any potential case confusion.